src/bshclient/src/main/java/org/apache/jmeter/util/BeanShellClient.java

```diff
@@ -80,7 +82,7 @@ public static void main(String [] args) throws Exception{
     private static void sendLine( String line, OutputStream outPipe )
     throws IOException
     {
-        outPipe.write( line.getBytes() ); // TODO - charset?
+        outPipe.write(line.getBytes(StandardCharsets.UTF_8));
```

Comment:
Are you sure about the potential breaks this might introduce ?

Reply:
I believe there's a movement to use UTF-8 by default.
* https://openjdk.java.net/jeps/8187041
* Kotlin uses UTF-8 by default

Of course, we can make it configurable, however, I believe, most of the times UTF-8 by default would be better (less chances to corrupt data) than use platform-default encoding.